ReactNativeClient/lib/joplin-renderer/MdToHtml/rules/copy_to_clipboard.js

```diff
@@ -0,0 +1,108 @@
+const fs = require('fs');
+const path = require('path');
+const filePath = path.join(__dirname, '../copyFunction.js');
+const btnCode = fs.readFileSync(filePath).toString();
+
+
+const copyCodeStyle = `
+.hljs {
+	position: relative;
+}
```

Comment:
Why does the hljs class need to be changed? If it does need to be changed, it should in noteStyle.js directly.

Reply:
i had to give it a ```position: relative```, to be able to to position the copy button with ```position:absolute```, but i'll take the style to noteStyle.js

Reply:
change completed